wherehows-frontend/conf/routes

```diff
@@ -286,6 +288,22 @@ GET           /api/v2/list/platforms
 
 GET           /api/v2/list/complianceDataTypes                                  controllers.api.v2.Dataset.getComplianceDataTypes
 
+GET           /api/v2/dataset/:urn                                              controllers.api.v2.Dataset.getDataset(urn: String)
```

Comment:
can we use plural for the `/dataset`, i.e. `/datasets` to be consistent with other v2 api's

Reply:
same for the below

Reply:
I was thinking for listing APIs, we use plural form, and for single item query, use singular form.